scheduler/src/cook/mesos/schema.clj

```diff
@@ -427,20 +441,20 @@
              - task succeeded => job completed
              - task failed, no other tasks, retries exceeded => job completed
              - task failed, no other tasks, retries remaining => job waiting
-             - task failed, other tasks running => job running"
+             - task failed, other tasks running => job running
+
+             Note that backfilled running instances are treated as if they don't exist."
```

Comment:
Also, can you update the comment to explicitly say that backfilled running instances will not move a job from waiting to running if that is the case.